advocacy_docs/partner_docs/QuestFoglightforDatabases/04-ConfiguringQuestFoglightforDatabases.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: 'Configuring Quest Foglight for Databases'
+description: 'Walkthrough on configuring Quest Foglight for Databases'
+---
+
+Implementing Quest Foglight for Databases with EDB Postgres Advanced Server and/or EDB Postgres Extended Server requires the following components:
+
+- EDB Postgres Advanced Server and/or EDB Postgres Extended Server
```

Comment:
"and/or" should just be "or"

Reply:
Jasmer, I think we are ok with the and/or as this product can monitor one or both of our servers at the same time.  It all depends on your environment.